apps/src/aichat/redux/thunks/publishModelCard.ts

```diff
@@ -0,0 +1,23 @@
+import {createAsyncThunk} from '@reduxjs/toolkit';
+
+import {RootState} from '@cdo/apps/types/redux';
+import {AppDispatch} from '@cdo/apps/util/reduxHooks';
+
+import {setModelCardProperty} from '../slice';
+
+import {saveAiCustomization} from './helpers/saveAiCustomization';
+
+// This thunk is used when a student fills out a model card and "publishes" their model,
+// enabling access to a "presentation view" where they can interact with their model
+// and view its details (temperature, system prompt, etc) in a summary view.
+export const publishModelCard = createAsyncThunk(
```

Comment:
Notice that this used to be called `publishModel` but think it makes sense to call it `publishModelCard`.

Reply:
Yep good catch! Renamed for clarity (forgot to include in description)